### ISSUE TITLE
Add login button to "Permission screen"

### ISSUE THREAD

**Issue by jasonLaster** [replayio/devtools#6637]
> i've logged in a couple of times using a wrong one, and it's annoying to get back to the main app~ screen to log out from there and go through the whole log in process again, in such a case usually i'm just log in from the app screen itself so im not even taken back immediately to that recording that i was about to access

![image](https://user-images.githubusercontent.com/254562/166307708-d1597ccf-c10f-4d22-959c-ba7731c9df15.png)

**Comment by jaril** [comment]
This has been brought up a bunch of times before. There's no escape hatch from the no-permissions error.

I used to hand-wave it away because it seems like the kind of problem that would only happen to internal folks, because we may use more than one email for replay. But recently we've also heard this feedback from other users. Turns out signing in with the wrong account is a common problem for anybody using oauth and has more than one google account.

Assigning it to May, but no sprint for now. @souporserious this might be a fun one to pick up since you'll be introduced to the _wonderful_ world of how we handle errors on the app. :)
